src/pages/designing/figma-kit.mdx

```diff
@@ -5,22 +5,162 @@ description: The new Figma design kit contains type styles, color tokens, and co
 
 <PageDescription>
 
-The Carbon for IBM.com team has been developing a Figma design kit that contains
-type styles, color tokens, and components.
+The Carbon for IBM.com Figma design kit contains type styles, color tokens, and components.
 
 </PageDescription>
 
-## Install the Figma kit
+<AnchorLinks>
+
+<AnchorLink>Accessing the new Figma kit</AnchorLink>
+<AnchorLink>Using the Figma kit</AnchorLink>
+<AnchorLink>Other Figma resources</AnchorLink>
+
+</AnchorLinks>
+
+## Accessing the new Figma kit
 
 <Row>
 <Column  colLg={8} colMd={8} colSm={4}>
 
-The Carbon for IBM.com team is alpha testing a Figma kit that contains type
-styles and all color tokens in the one kit.
+We have a new Figma kit for delivering pre-built versions of the Carbon for IBM.com components. The kit contains libraries for each theme and includes color tokens, expressive and productive type styles, components, and grid templates.
+
+We’ll update the kit with more components and adjustment to existing components over time, and Figma will notify you when library updates are available.
```

Comment:
I think this sentence might be better using what's existing on Sketch page. "adjustment to existing components" doesn't sound right: 

> We’ll be adding more components over time, and Figma will notify you when library updates are available.